xpcPlugin/MessageHandlers.h

```diff
@@ -54,6 +57,14 @@ namespace XPC
 
 		static void HandleXPlaneData(const Message& msg);
 		static void HandleUnknown(const Message& msg);
+		
+		static int CamFunc( XPLMCameraPosition_t * outCameraPosition, int inIsLosingControl, void *inRefcon);
+		
+		struct camera_properties{
```

Comment:
Rename to `CameraProperties` to be consistent with project naming conventions

Reply:
fixed